launch

```diff
@@ -162,15 +163,25 @@ if __name__ == '__main__':
         dbg_cmd = shlex.split(DEBUGGERS[args.debugger] %
                               {'kernel': args.kernel, 'triplet': triplet})
 
+        # start_new_session creates new process group in fresh session,
+        # so when the terminal emulator sends a signal on CTRL+C,
+        # SIGINT will not propagate it to *sim* and *uart* processes
         sim = subprocess.Popen(sim_cmd, start_new_session=True)
-        gdb = subprocess.Popen(dbg_cmd, start_new_session=True)
+        # FIXME Because popen_spawn.PopenSpawn cannot accept start_new_session,
+        # we need to resort to sleep synchronization :(
+        time.sleep(1)
```

Comment:
I suppose this `sleep` might be moved to inside `xterm`, which would achieve the same result, while making the script seem more responsive, as the debugger could start earlier - it's only `nc` who needs to wait.